tasks/ostree-boot-image

```diff
@@ -1,10 +1,23 @@
 #!/bin/sh
 
-set -x
+set -xe
 env
 
 # Find the base dir
 base_dir="$(dirname $0)/.."
+HOMEDIR=$(pwd)
+
+function clean_up {
+    set +e
+    # Tear down the atomic host and image
+    sudo docker run --privileged -it --link libvirtd:libvirtd -e commit=${commit} -e image2boot=${image2boot} -e state=absent -v /var/lib/libvirt/:/var/lib/libvirt/ -v ${HOMEDIR}/output/:/root/output/ -v ${basedir}/:/root/ci-pipeline ostree_boot_image
```

Comment:
So this is the part that I think will not be part of this stage and will instead come after the integration and package level tests once this is put into container() calls in the Jenkinsfile

Reply:
agreed.  But for now we can do it here.